test/programs/pizza-ordering/cli/entry.js

```diff
@@ -1,4 +1,6 @@
 // Initialize Waterfall CLI
+console.log('entry.js: __dirname = ' + __dirname);
```

Comment:
Why was this added and verbose changed to `false`?

Reply:
It was added for debugging purposes.  It can be removed now.  Same for the verbose: false.